### PR TITLE
Update VST path

### PR DESCRIPTION
**Pull request by hfiguiere**
Old path will be deprecated in 22.08

### Description

In Flatpak vst plugins are moving. All current plugin in 21.08 series can use both. In 22.08, this will change.

### Motivation and Context

When you move to a 22.08 based runtime, things will break.

### How Has This Been Tested?

Hasn't been test per see, I know the changes. The only question is how does OBS remember of plugins? By their ID or by their PATH?

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [ ] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

File: CI/flatpak/com.obsproject.Studio.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "--talk-name=org.a11y.Bus",
     "--own-name=org.kde.StatusNotifierItem-2-2",
     "--system-talk-name=org.freedesktop.Avahi",
-    "--env=VST_PATH=/app/extensions/Plugins/lxvst"
+    "--env=VST_PATH=/app/extensions/Plugins/vst"
```
Comment (tytan652) [inline CI/flatpak/com.obsproject.Studio.json:22]:
```suggestion
    "--env=VST_PATH=/app/extensions/Plugins/lxvst;/app/extensions/Plugins/vst"
```

File: CI/flatpak/com.obsproject.Studio.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
       "directory": "extensions/Plugins",
       "version": "21.08",
       "add-ld-path": "lib",
-      "merge-dirs": "lxvst",
+      "merge-dirs": "vst",
```
Comment (tytan652) [inline CI/flatpak/com.obsproject.Studio.json:37]:
```suggestion
      "merge-dirs": "lxvst;vst",
```

**Review by tytan652 — COMMENTED** [review]
The commit is not named correctly. Use forced push, don't create a new PR.
And please verify the checkboxes in the PR description.

About the changes, since we can have multiple path in `VST_PATH`, just adding it and not replacing would be better.

**Comment by hfiguiere** [comment]
> About the changes, since we can have multiple path in `VST_PATH`, just adding it and not replacing would be better.

No it would NOT.

**Comment by hfiguiere** [comment]
> You said that both were used in 21.08, we are not going to suddenly switch to 22.08 when available.

I said the were changing and both are available - for a transition. But not that they should be used simultaneously.

Now if you want to not change that know and have it break because you forget when moving to a 22.08 based runtime, it's up to you. Given that OBS flatpak manifest isn't hosted in the flathub organisation I don't have the possibility to fix this myself.

**Comment by tytan652** [comment]
> Now if you want to not change that know and have it break because you forget when moving to a 22.08 based runtime, it's up to you.

I meant by that, that we have time before we switch to 22.08.

This change is needed to avoid a breakage in the future, but this PR can't be merged if you don't comply to the contributing document.

**Pull request closed without merging**